src/page-objects/storefront/AccountOrder.ts

```diff
@@ -8,7 +8,7 @@ export class AccountOrder implements PageObject {
     public readonly digitalProductDownloadButton: Locator;
 
     constructor(public readonly page: Page) {
-        this.orderExpandButton = page.getByRole('button', {name: 'Expand'}).first();
+        this.orderExpandButton = page.locator('.order-hide-btn').first();
```

Comment:
Usually, we want to prefer aria selectors (+ text) instead of classes. You can use a regex to match the old and new string.

Reply:
Ah nice trick, didn't know that this is possible here. 👍